### ISSUE TITLE
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table, in Admin screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#884]
**Describe the bug**
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**To Reproduce**

1. Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test&path=%2FSites%2Ftest%2Findex.html&pathType=page)
2. Click on "Configure step" button given in right top side of Draft box, under Default Workflow.
3. It will open a pop up, with a table, for Configuring step.
4. Inspect element and you will find that scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**Expected behavior**
scope="col" for th tag and scope = "row" for very first td in each tr tag, in table, should be present.

**Additional context**
Navigate all the options available for similar issue and fix.
